scripts/create-merged-schema.ts

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ts-node
```

Comment:
I'm assuming this line doesn't work, since we use `node --require` in the script?

Reply:
It should work if you have `ts-node` installed globally, but that's not technically true all the time, so I put `node --require` in the package script.